docs/running-on-yarn.md

```diff
@@ -304,6 +304,16 @@ If you need a reference to the proper location to put log files in the YARN so t
   </td>
 </tr>
 <tr>
+  <td><code>spark.yarn.am.attemptFailuresValidityInterval</code></td>
+  <td>(none)</td>
+  <td>
+  Defines the validity interval (in millisecond) for AM failure tracking.
+  If the AM has been running for at least long, the AM failure count will be reset.
+  Default value None means this validity interval is not enabled.
```

Comment:
There is no default value. This should say: "If not set, the feature is not enabled."